frontend/src/app/containers/ExperimentPage/ExperimentTourDialog.js

```diff
@@ -48,7 +49,10 @@ export default class ExperimentTourDialog extends React.Component {
   }
 
   renderHeaderTitle = () => {
-    const { experiment, isExperimentEnabled } = this.props;
+    const { experiment, isExperimentEnabled, isFirefox } = this.props;
+    if (!isFirefox) {
```

Comment:
The user can also show the tour if they are using Firefox but don't have the experiment enabled, so this should actually check !isExperimentEnabled.